app/javascript/article-form/components/Help/index.jsx

```diff
@@ -0,0 +1,122 @@
+import { h, Component } from 'preact';
+import PropTypes from 'prop-types';
+import ArticleFormTitle from './ArticleFormTitle';
+import TagInput from './TagInput';
+import BasicEditor from './BasicEditor';
+import Format from './Format';
+import { Modal } from '@crayons';
+
+export class Help extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      liquidHelpHTML:
+        document.getElementById('editor-liquid-help') &&
+        document.getElementById('editor-liquid-help').innerHTML,
+      markdownHelpHTML:
+        document.getElementById('editor-markdown-help') &&
+        document.getElementById('editor-markdown-help').innerHTML,
+      frontmatterHelpHTML:
+        document.getElementById('editor-frontmatter-help') &&
+        document.getElementById('editor-frontmatter-help').innerHTML,
+    };
+  }
+
+  setCommonProps = ({
+    liquidShowing = false,
+    markdownShowing = false,
+    frontmatterShowing = false,
+  }) => {
+    return {
+      liquidShowing,
+      markdownShowing,
+      frontmatterShowing,
+    };
+  };
+
+  toggleModal = (varShowing) => (e) => {
+    e.preventDefault();
+    this.setState((prevState) => ({
+      ...this.setCommonProps({
+        [varShowing]: !prevState[varShowing],
+      }),
+    }));
```

Comment:
refactor (blocking): `setCommonProps` is only ever called to spread the intial state of some state properties in `toggleModal`. Why not make it part of the initial state and then you can use `prevState` to get the value you need to set the key `[varShiowing]`.

```suggestion
    this.setState((prevState) => ({
      [varShowing]: !prevState[varShowing],
    }));
```